src/main.rs

```diff
@@ -39,7 +39,9 @@ pub async fn execute_subcommand(command: Option<Command>) -> ResultWithDefaultEr
         None => RunningTimeEntryCommand::execute(get_api_client()?).await?,
         Some(subcommand) => match subcommand {
             Stop => StopCommand::execute(get_api_client()?).await?,
-            Continue => ContinueCommand::execute(get_api_client()?).await?,
+            Continue { interactive } => {
+                ContinueCommand::execute(get_api_client()?, interactive).await?
+            }
```

Comment:
Are these braces added by clippy? IMO It looks better without them

Reply:
Clippy cause line length go wack,
we can tweak the config if you want though?

Personally I like not having any config and let the default standard guide us.

Reply:
I also prefer no configuration, so let's leave it at that